lib/lte_link_control/lte_lc.c

```diff
@@ -545,14 +530,12 @@ static int enable_notifications(void)
 		 * considered a critical error, and the error code is therefore
 		 * not returned, while informative log messageas are printed.
 		 */
-		LOG_WRN("%s failed (%d), RRC notifications are not enabled",
-			cscon, err);
-		LOG_WRN("%s is supported in nRF9160 modem >= v1.1.0", cscon);
+		LOG_WRN("AT+CSCON failed (%d), RRC notifications are not enabled", err);
+		LOG_WRN("AT+CSCON is supported in nRF9160 modem >= v1.1.0");
 
-		err = at_cmd_write("AT+CGMR", buf, sizeof(buf), NULL);
+		err = nrf_modem_at_cmd(buf, sizeof(buf), "AT+CGMR");
 		if (err == 0) {
-			LOG_WRN("Current modem firmware version: %s",
-				log_strdup(buf));
+			LOG_WRN("Current modem firmware version: %s", log_strdup(buf));
```

Comment:
`nrf_modem_at_cmd()` returns the whole modem response, which includes "OK" in addition to the firmware version. So the output is like this:

```
[00:00:00.288,421] <wrn> lte_lc: Current modem firmware version: mfw_nrf9160_1.3.0
OK
```

Reply:
Ah yeah, forgot about that... Doesn't look fantastic, so I'll terminate the string before the OK

Reply:
Fixed now